editor/reducer.js

```diff
@@ -657,6 +658,13 @@ export function metaBoxes( state = defaultMetaBoxState, action ) {
 	}
 }
 
+// Create responsive reducer with lib default breakpoints excluding small where we are currently using 782.
+const responsive = createResponsiveStateReducer( {
```

Comment:
Should we use variables imported from `.scss` file instead?

Reply:
Yes, that's the plan, but the PR that adds an scss importer https://github.com/WordPress/gutenberg/pull/3332 is not yet merged. It would be perfect if we can a new review on it and land it before this PR so we can make use of the importer :)

Reply:
You referenced wrong PR number :)

Reply:
Yes, you are right, the correct PR number is https://github.com/WordPress/gutenberg/pull/3331. (comment was updated)